### ISSUE TITLE
Lighthouse timings are longer than the run time of Lighthouse

### ISSUE THREAD

**Issue by bryanhickerson** [GoogleChrome/lighthouse#11000]
#### Provide the steps to reproduce
1. Run Lighthouse in Chrome dev tools or via cli when authenticated to my app

I cannot provide access to a repo, but I could privately provide test credentials to our app.

#### What is the current behavior?
Run Lighthouse in Chrome dev tools, and various results are over 10 seconds despite it only taking 10 seconds for Lighthouse to run end to end. It seems to be underreporting performance, which makes it challenging to use as a tool to gauge performance impact.

[app.betterup.co-20200620T113014.txt](https://github.com/GoogleChrome/lighthouse/files/4808393/app.betterup.co-20200620T113014.txt)

#### What is the expected behavior?
That the runtime of lighthouse would be upper limit of the scores provided.

#### Environment Information
* Affected Channels: DevTools 
* Lighthouse version: 5.7.1
* Chrome version: 83.0.4103.106 (Official Build) (64-bit)
* Node.js version: 12.16.2
* Operating System: macOS Catalina 10.15.5

**Related issues**
None that I could find.

**Comment by patrickhulce** [comment]
Thanks for filing @bryanhickerson! This is covered in our [FAQs](https://github.com/GoogleChrome/lighthouse#why-is-the-performance-score-so-low-it-looks-fine-to-me) and [throttling documentation](https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md).

tl;dr - use *applied* throttling if you want to watch Lighthouse load it slowly

**Comment by bryanhickerson** [comment]
Thanks for bringing this to my attention. I wasn't aware of this setting. Unfortunately, the scoring still seems incongruous with actual page load times, but I'll continue digging.

**Comment by bryanhickerson** [comment]
Found the option I was looking for via cli: `--throttling-method=provided`
